pkgs/tools/filesystems/afpfs-fuse/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, libgcrypt, gmp, readline, fuse }:
+
+stdenv.mkDerivation rec {
+  name = "afpfs-ng-fuse-0.8.2";
```

Comment:
I think it's somewhat unfortunate that the derivation name (sans version) differs from the file name and top-level attribute name.